internal/service/project/data_source_project.go

```diff
@@ -137,12 +138,40 @@ func (d *projectDS) Schema(ctx context.Context, req datasource.SchemaRequest, re
 					},
 				},
 			},
+			"ip_addresses": schema.SingleNestedAttribute{
+				Attributes: map[string]schema.Attribute{
+					"services": schema.SingleNestedAttribute{
+						Attributes: map[string]schema.Attribute{
+							"clusters": schema.ListNestedAttribute{
+								NestedObject: schema.NestedAttributeObject{
```

Comment:
it seems like IP addresses are associated to clusters in the project?   wouldn't it make more sense then that IP addresses are returned in the clustest/adv_cluster resource instead of project?   or in addition to be returned in project?

Reply:
This a fair point. The main reason I see for including in the project resource is due to the endpoint being tagged in the project section (https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Projects/operation/returnAllIPAddresses).